eth/catalyst/api.go

```diff
@@ -405,8 +293,5 @@ func (api *ConsensusAPI) setHead(newHead common.Hash) error {
 
 // Helper function, return the merger instance.
 func (api *ConsensusAPI) merger() *consensus.Merger {
```

Comment:
I kinda like this, because we have less code, but we could inline this

Reply:
yup